packages/shared/locales/en.json

```diff
@@ -260,6 +260,16 @@
                 "title": "General",
                 "description": "Configure your app's appearance and other general settings"
             },
+            "profile": {
+                "title": "Profile",
+                "description": "Change your profile name or set an avatar"
```

Comment:
Are we using this description?

Reply:
Not sure where this is used.. I just moved its location in the en file.